src/jest/index.ts

```diff
@@ -0,0 +1 @@
+import 'jest';
```

Comment:
what's happening with this file?

Reply:
this adds global jest declarations to the module. the better way to do that would be to have two different tsconfig where one defines "jest" in the "types"  config, and two different compilation outputs. Sounds like a lerna thing, but I think it's so overkill

Reply:
let me make this a d.ts so it really doesn't get compiled